lib/util/string.js

```diff
@@ -19,6 +19,14 @@
  * limitations under the License.
  */
 
-/**
- * @typedef {Request} KuzzleRequest
- */
+const Request = require('kuzzle-common-objects').Request;
+
+module.exports = {
+  /**
+   * @param {Request} request
+   * @returns {string}
+   */
+  getRequestRoute(request) {
```

Comment:
I agree: there is no added value to adding this piece of information to the error messages since clients already have it in their request object.